docs/content/installation/migration/ingress-nginx.md

```diff
@@ -0,0 +1,549 @@
+| [`nginx.ingress.kubernetes.io/server-snippet`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#server-snippet)| [`nginx.org/server-snippets`](https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/#snippets-and-custom-templates) | N/A |
+| [`nginx.ingress.kubernetes.io/ssl-redirect`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#server-side-https-enforcement-through-redirect) | [`ingress.kubernetes.io/ssl-redirect`](https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/#auth-and-ssltls) | N/A (2) |
+{{% /bootstrap-table %}}
+
+1. Ingress-NGINX Controller implements some of its load balancing algorithms with Lua, which may not have an equivalent in NGINX Ingress Controller.
+1. To redirecting HTTP (80) traffic to HTTPS (443) NGINX Ingress Controller uses native NGINX `if` conditions while Ingress-NGINX Controller uses Lua.
+
+The following two snippets outline what Ingress-NGINX Controller annotations correspond to annotations for NGINX Ingress Controller with NGINX Plus.
+
+**Ingress-NGINX Controller**
+```yaml
+nginx.ingress.kubernetes.io/affinity: "cookie"
+nginx.ingress.kubernetes.io/session-cookie-name: "cookie_name"
+nginx.ingress.kubernetes.io/session-cookie-expires: "seconds"
+nginx.ingress.kubernetes.io/session-cookie-path: "/route"
+```
+
+**NGINX Ingress Controller (with NGINX Plus)**
+```yaml
+nginx.com/sticky-cookie-services: "serviceName=example-svc cookie_name expires=time path=/route"
+```
+
+{{< note >}}
+The NGINX Ingress Controller has additional annotations for features using NGINX Plus which have no Ingress-NGINX Controller equivalent, such as active health checks and authentication using JSON Web Tokens (JWTs).
+{{< /note >}}
+
+### Global Configuration with ConfigMaps
+
+This table maps the Ingress-NGINX Controller ConfigMap keys to NGINX Ingress Controller's equivalent ConfigMap keys.
+
+<!-- {{< note >}}
+Some of the key names are identical, and each Ingress Controller has ConfigMap keys that the other does not (Which are indicated).
+{{< /note >}} -->
+
+{{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
+| Ingress-NGINX Controller | NGINX Ingress Controller |
+| ------------------------ | ------------------------ |
+| [`disable-access-log`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#disable-access-log) | [`access-log-off`](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/configmap-resource/#logging) |
```

Comment:
Same question as above -- should these be formatted as in-line code or bold or italic?

Reply:
Same answer as above!

I've formatted these all as in-line code to march the original blog post.

https://www.nginx.com/blog/migrating-from-community-ingress-controller-to-f5-nginx-ingress-controller/

Reply:
The blog post isn't a style guide. I think we decided on bold? I raised the issue in Slack.